projects/components/src/color-picker/color-picker.component.ts

```diff
@@ -41,6 +46,9 @@ export class ColorPickerComponent implements ControlValueAccessor, OnChanges {
   @Input()
   public selected?: string;
 
+  @Input()
+  public disabled: boolean = false;
```

Comment:
There is alway a risk of the user passing in a `null` or `undefined` here, so it's safest to type this as optional `?`.

Reply:
done